test/pipelines-it-remote.yml

```diff
@@ -0,0 +1,59 @@
+jobs:
+- job: 'integration_test_remote'
+  pool: 'NNI CI KUBE CLI'
```

Comment:
The remote CI's agent pool is shared with KUBE?

Reply:
removed in yaml file.